server/blob/blob-deduplicating/src/test/scala/GCPropertiesTest.scala

```diff
@@ -127,7 +141,7 @@ object Generators {
 
 
 object GCPropertiesTest extends Properties("GC") {
-  implicit val arbEvents: Arbitrary[Seq[Event]] = Arbitrary(Generators.eventsGen)
+  val arbEvents: Arbitrary[Seq[Event]] = Arbitrary(Generators.eventsGen(maxNbEvents = 100, maxHashReuseRatio = 0.2f))
```

Comment:
1000 is working, but 10 000 is still not finished after almost one hour on my computer. Any idea why?